moja-kuhinjica-frontend/src/components/modal/login/LoginModal.tsx

```diff
@@ -18,31 +23,49 @@ export interface RegistrationFormFields {
 export const LoginModal = ({
     modalIsOpen,
     closeModal,
+    loggedIn,
+    setLoggedIn,
 }: ILoginModalProps): JSX.Element => {
+    const [showError, setShowError] = useState<boolean>(false)
     const {
         register,
         handleSubmit,
         reset,
         formState: { errors },
     } = useForm()
-    const onSubmit = (data: any): void => {
-        reset()
+
+    const login = (inputData: FieldValues): void => {
+        setShowError(false)
+        const res = UserService.login(inputData)
```

Comment:
Why do you use here `res`, when it is not used anywhere

Reply:
Removed.